ui/app/routes/vault/cluster/clients/index.js

```diff
@@ -11,6 +11,7 @@ const getActivityParams = ({ tab, start, end }) => {
   if (tab === 'current') {
     params.tab = tab;
   } else if (tab === 'history') {
```

Comment:
Just to clarify - this whole `tab === 'history'` block will be replaced by the `dashboard` block

Reply:
correct. this can actually be removed, but I'm keeping it here so I can remember the remove the query param work as well. However, I don't want to do that until we've gotten the dashboard component working with the 1.9 payload. so TLDR; it will be removed, soon just not in this pr.